src/controller/PostController.php

```diff
@@ -32,21 +35,27 @@
 class PostController extends AbstractController
 {
     /**
-     * Summary of template
+     * Summary of _instance
      * 
-     * @var TemplateInterface
+     * @var PostController
      */
-    public TemplateInterface $template;
+    private static $_instance;
 
     /**
-     * Summary of _instance
+     * Summary of _postService
      * 
-     * @var PostController
+     * @var PostService
      */
-    private static $_instance;
+    private $_postService;
```

Comment:
```suggestion
    private PostService $_postService;
```

Attention PHP 8 ! on type les variables en Php.

Reply:
fait